### PR TITLE
Twopart

### PR DESCRIPTION
**Pull request by ktpolanski**
Reorganise the readme to be a more gradual introduction to the world of TS. Add bits and pieces like the git remote thing that felt pertinent.

### CODE REVIEW

**Comment by docrostov** [comment]
overall i like it. some initial thoughts:

- the chest hair thing is meh to me, i am sorry. let's please try to not do weird hyper-macho jokes, i am fine with adding humor but we should do so in a way that is inclusive rather than masculine-leaning IMO. maybe like... "part one: hacking the mainframe", "part two: the mainframe hacks you", "part three: you have married the mainframe". this is bad but we can do better than chest hair. maybe increasingly elaborate pizza styles?
- i think given the goal of this is to try and de-obfuscate some of the weirdness, we probably need to explicitly address things like "what is babel/webpack" in step 1 and "what is node.js". perhaps a glossary that is linked throughout with term definitions? i would also probably link to the ["how to use the terminal"](https://loathing-associates-scripting-society.github.io/KoL-Scripting-Resources//PR-Overview.html#commandPrompt) guide on the garfjalen. (that also includes some vscode links we may want to include here)
- we should add more screenshots i think. screenshots are very good in guides like this. as we make this more detailed and involved we should also make it easier to follow through that. i am a little sad that my originally 7 step guide is now this long but i guess that's TS for you.

i can do some other spot edits to the grammar/wording another time, wanted to at least start the convo on this stuff.

**Comment by ktpolanski** [comment]
I agree that explaining things like babel/webpack/node.js is good. Unfortunately as I'm super green I won't be of much use. I would suggest doing so within the point itself to avoid having the reader jump around the place.

The only part I could see potentially benefitting from screenshots would be the copying of the scripts to the mafia folder. But I tend to prefer to take in information as text.

File: README.md [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
   }
 }
 
-export function main() {
+export function main(): void {
   print(checkMP());
 }
 ```
+- A helpful command within `yarn` is `yarn init`. This is useful for changing details about the package without needing to manually muck around `packages.json` and hoping you changed everything correctly. Call `yarn init` now, and provide a cool name for your package (without any spaces). You will be then asked further questions like what version of the package this is, a description of said package and so on. You can skip any you don't feel like answering by pressing enter.
+- Repeat the `yarn run build` from last time, and this time the scripts folder will have a subfolder named whatever you named the project. Neat!
```
Comment (lewismd13) [inline README.md]:
```suggestion
- Repeat the `yarn run build` from last time, and this time the scripts folder will have a subfolder named whatever you named the project. Neat! As you are editing, you can also call `yarn watch` and your script will automatically build on save, so you don't have to keep typing `yarn run build` every few minutes.
```

**Review by lewismd13 — CHANGES_REQUESTED** [review]

**Comment by ktpolanski** [comment]
Think that's the Manny notes sorted, with a bit of my own twist to some of this stuff. I agree forking is conceptually preferable, but I also remember disregarding the fork instruction and just doing a clone as I was trying to get TS going and wasn't expecting to do anything relevant with the kiddie pool repo :P

**Pull request merged**